Source/IntegrationTests/LitTests.cs

```diff
@@ -535,7 +543,7 @@ class MultiBackendLitCommand : ILitCommand {
     public (int, string, string) Execute(TextReader inputReader,
       TextWriter outputWriter,
       TextWriter errorWriter) {
-      var exitCode = new MultiBackendTest(inputReader, outputWriter, errorWriter).Start(arguments.Prepend("for-each-compiler"));
+      var exitCode = new MultiBackendTest(inputReader, outputWriter, errorWriter).Start(arguments);
```

Comment:
I assume you found this because of actually running tests with DAFNY_INTEGRATION_TESTS_IN_PROCESS=true? Thanks for the fix, and we should add some singleton tests for that mode to protect against regressions (not as part of this PR)